hw/ip/prim/util/sparse-fsm-encode.py

```diff
@@ -30,6 +31,22 @@
 USAGE = "./sparse-fsm-encode -m <#states> -n <#bits> -d <minimum HD> -s <seed>"
```

Comment:
nit: can you update the usage string as well?

Reply:
Done, using argparse's support for this rather than a hard-coded string.